### PR TITLE
Station Blueprint Hotfix

### PR DESCRIPTION
**Pull request by Geevies**
* Station Blueprints now count asteroid turf as space, this should reduce server crashing when using them.

### CODE REVIEW

**Comment by Geevies** [comment]
!bugfix

**Comment by Geevies** [comment]
!review

File: code/game/objects/items/blueprints.dm [hunk 1]
```diff
@@ -19,22 +19,22 @@
...
+	if(use_check_and_message(user, USE_DISALLOW_SILICONS))
 		return
 	add_fingerprint(user)
 	interact()
 	return
 
 /obj/item/blueprints/Topic(href, href_list)
 	..()
-	if ((usr.restrained() || usr.stat || usr.get_active_hand() != src))
+	if((usr.restrained() || usr.stat || usr.get_active_hand() != src))
```
Comment (Arrow768) [inline code/game/objects/items/blueprints.dm:31]: Is there a reason you arnt using `use_check_and_message` or` use_check `here?

Comment (Geevies) [reply code/game/objects/items/blueprints.dm]: updated

**Comment by Geevies** [comment]
> 
> 
> You most likely have to update `/obj/item/blueprints/proc/get_area()` aswell
> (Possibly by adding a new AREA_ASTEROID)

it already checks /area/mining/unexplored in get_area_type, so i take it that that's fine

**Comment by skull132** [comment]
Can you also fix APC and air alarm placement? They can currently be placed down on the asteroid, resulting in the server keeling.

**Pull request merged**